data/heliarch intro.txt

```diff
@@ -0,0 +1,1568 @@
+			choice
+				`	(Check the warehouses with the rusty doors.)`
+					goto rustyending
+				`	(Check the warehouses with infiltration.)`
+					goto reactor
+			label rustyending
+			`	Kirrie uses his card to open the door, and continues to explain that it is an access card granted to Heliarch agents, in case they need to do something like this, or just identify themselves as one.`
+			`	Sadly the warehouse turns out to not contain anything significant. It's a storage for what Kirrie says are toy parts, just mountains of plastic stacked together.`
+			`	You check a few more like this, but none turn out to have anything, and as the rain and wind become too much to walk around, you take shelter in one of the warehouses.`
+			`	Once the weather settles, near nightfall, he gets notified by Iplora that she and Tugroob are already back at the ship, so you two decide to head back as well.`
+			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
+				accept
+			label trees2
+			`	You suggest looking for something unusual near the plant life you spotted, while the rain is still fairly tame, and he follows you. You spend a few minutes looking around the trees, but find nothing suspicious.`
+			`	While passing by another set of bushes, he notices that some of them are awkwardly tilted. Upon further inspection, he tells you that they appear to have been removed, and then forcibly put on the ground again, noting some odd, wavy patterns of the small rocks around the roots. The rain gets even heavier.`
+			choice
+				`	(Dig out the bushes.)`
+					goto bushending
+				`	(Head back to the open warehouses and take shelter there.)`
+					goto beetledigger
+			label beetledigger
+			`	Though you can barely hear each other in the rain, Kirrie insists in staying there and digging through the ground for a while, to see if he finds anything.`
+			`	You walk back to the empty streets, taking care to not slip on the damp soil, and enter one of the warehouses Kirrie had previously opened. After a few minutes, he walks in, saying he dug out many, but couldn't find anything under the sets of bushes.`
+			`	Once the weather settles, near nightfall, he gets notified by Iplora that she and Tugroob are already back at the ship, so you two decide to head back as well.`
+			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
+				accept
+			label lonebeetle
+			`	He leaves too, and you're on your own in the ship for a couple hours. Eventually the get back, and it seems only Tugroob found something, mentioning some evidence of smuggled weapons passing through the planet.`
```

Comment:
`	Kirrie leaves you behind and you sit there on your own for a number of hours. Eventually, the trio returns, revealing that only Tugroob had found something, mentioning some evidence of smuggled weapons passing through the planet.`